crates/services/p2p/src/peer_manager.rs

```diff
@@ -291,22 +295,11 @@ impl PeerManager {
             .send(self.reserved_connected_peers.len());
     }
 
-    fn insert_peer_info(&mut self, peer_id: &PeerId, data: PeerInfoInsert) {
-        let peers = if self.reserved_peers.contains(peer_id) {
+    fn get_relevant_peers(&mut self, peer_id: &PeerId) -> &mut HashMap<PeerId, PeerInfo> {
```

Comment:
Nit: maybe this could be renamed to `relevant_peers_mut`, since it returns mutable access to underlying data?